plugin/storage/integration/elasticsearch_test.go

```diff
@@ -62,11 +61,9 @@ const (
 
 type ESStorageIntegration struct {
 	StorageIntegration
-
-	client        *elastic.Client
-	v8Client      *elasticsearch8.Client
 	bulkProcessor *elastic.BulkProcessor
 	logger        *zap.Logger
+	*EsClient
```

Comment:
```suggestion
	client *EsClient
```
only embed structs if you want the outer struct TO BE like the embedded struct. This is clearly not the case - you don't want ESStorageIntegration to be an ES Client. So make it a proper named field.